app/grandchallenge/reader_studies/forms.py

```diff
@@ -470,3 +471,25 @@ def clean_ground_truth(self):
         values = [x for x in rdr]
 
         return values
+
+
+class AddCasesForm(UploadRawImagesForm):
```

Comment:
This new form should have a test, is it only the url in the help text that is changing? Then maybe add that to the forms init kwarg and use the same form in both places.